docs/api/phpdoc/classes/PhlyRestfully.Resource.html

```diff
@@ -258,13 +258,13 @@
 <a id="method_getRouteMatch"></a><div class="element clickable method public method_getRouteMatch" data-toggle="collapse" data-target=".method_getRouteMatch .collapse">
 <h2>getRouteMatch()
         </h2>
-<pre>getRouteMatch() : null | \Zend\Mvc\Router\RouteMatch</pre>
```

Comment:
Note to self: remove the `docs/api/` tree, as it's completely out-of-date! (References PhlyRestfully!)